linux_os/guide/system/bootloader-grub2/uefi/grub2_uefi_password/oval/shared.xml

```diff
@@ -3,19 +3,20 @@
     {{{ oval_metadata("The UEFI grub2 boot loader should have password protection enabled.") }}}
 
     <criteria operator="OR">
-      {{{ oval_file_absent_criterion(grub2_uefi_boot_path + "/grub.cfg") }}}
+      {{% if product in ["ol8", "rhel8"] %}}
```

Comment:
Why is RHEL 8 and OL 8 different? Does it apply to other Linux distributions as well?

Reply:
I hadn't reviewed other distros. This change is based on DISA's STIG profile documentation, and apparently this also applies to el7. Do you want me to make the change to apply to all products?

This rule is used only in STIG profiles in rhel and ol

Reply:
OK, thanks for reply. When I read the rationale that the criteria is now handled by applicability check (grub cpe) I think that this rationale would apply for any distribution. So please remove the conditions and make the change apply on all products. As you say the rule is used only in RHEL and OL STIG I feel that it's a conservative change.